examples/rails/080_configuration/.werf_secret_key

```diff
@@ -0,0 +1 @@
+aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
Why do we need this file? It is always a bad idea to store `.werf_secret_key` in the repository.